test/recipes/30-test_evp_data/evppkey_dsa.txt

```diff
@@ -339,6 +341,7 @@ Input = "Hello"
 Result = DIGESTSIGNINIT_ERROR
 
 # Test sign with a 3072 bit key with N == 224 is not allowed in fips mode
+FIPSversion = <3.4.0
```

Comment:
Keep this consistent.. move below availablein

Reply:
Done